### DISCUSSION TITLE
The docker exit code is always 1.

### DISCUSSION THREAD

**Discussion by ghost** [ryanoasis/nerd-fonts#1316] [Q&A]
```
$ docker run --rm -v ~/in:/in -v ~/out:/out nerdfonts/patcher -v
Running with options:
 -v
Copyright (c) 2000-2023. See AUTHORS for Contributors.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 with many parts BSD <http://fontforge.org/license.html>. Please read LICENSE.
 Version: 20230101
 Based on sources from 2023-04-28 21:13 UTC-D.
Nerd Fonts Patcher v3.0.1 (4.4.0) (ff 20230101)
Nerd Fonts: font-patcher (3.0.1)
$ echo $?
1
```

**Comment by Finii** [comment]
Thanks for reporting.

Will be fixed with the new container that is being built right now.

Ref: https://github.com/ryanoasis/nerd-fonts/commit/c173f661a0ed03bc537d31a79670bc03b586251d

Container ready...:

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/fa570bde-c0ee-42df-a50d-e4d34a18a68b)
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/5856884b-e79a-4ea6-9f2d-bccc45e2f6b8)
